CMakeLists.txt

```diff
@@ -333,6 +333,47 @@ else()
 endif()
 add_subdirectory(doc)
 
+## #################################################################
```

Comment:
It would be better to move this cmake script in a new cppyy/CMakeLists.txt
and do 
```
if(GISMO_WITH_CPPYY)
  add_subdirectory(cppyy) 
endif()
```
here.
This way the main CMakeLists.txt is more clear and also the cppyy related script more autonomous.